frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/items/BaseGraphItem.java

```diff
@@ -12,35 +12,35 @@
 
 @Getter
 @ToString
-public abstract class BaseGraphNode implements IGraphNode {
+public abstract class BaseGraphItem implements IGraphItem {
 
   private final GraphEdgeColor graphEdgeColor;
 
   @GTENegativeOne
-  private final int prevSiblingNodeIndex;
+  private final int prevSiblingItemIndex;
 
   @Positive
   @MonotonicNonNull
-  private Integer nextSiblingNodeIndex = null;
+  private Integer nextSiblingItemIndex = null;
 
   @NonNegative
   private final int indentLevel;
 
-  protected BaseGraphNode(GraphEdgeColor graphEdgeColor,
-      @GTENegativeOne int prevSiblingNodeIndex,
-      @Positive int nextSiblingNodeIndex,
+  protected BaseGraphItem(GraphEdgeColor graphEdgeColor,
```

Comment:
Would be nice to specify **what** edge is meant here... edge to parent?

Reply:
Hmmm :thinking:  good point. It seems like this name was not accurate in the first place since it was added.

Reply:
renamed to `GraphItemColor`